src/primitives/block.cpp

```diff
@@ -12,7 +12,7 @@
 
 uint256 CBlockHeader::GetHash() const
 {
-    return SerializeHash(*this);
+    return (CHashWriter(PROTOCOL_VERSION) << *this).GetHash();
```

Comment:
Note this was the only call that relied on the default `SerializeHash` `nVersion` value.